examples/default/variables.tf

```diff
@@ -19,7 +19,7 @@ variable "region" {
 variable "prefix" {
   description = "The prefix that you would like to append to your resources"
   type        = string
-  default     = "test-landing-zone-vsi"
+  default     = "rajat-vsi"
```

Comment:
dont use `rajat` in the prefix default value

Reply:
yeah was running tests locally.